crates/aptos-api-tester/src/main.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+#![forbid(unsafe_code)]
+
+mod consts;
+mod counters;
+mod persistent_check;
+mod strings;
+mod tests;
+mod token_client;
+mod utils;
+#[macro_use]
+mod macros;
+
+use crate::utils::{NetworkName, TestName};
+use anyhow::Result;
+use aptos_logger::{info, Level, Logger};
+use aptos_push_metrics::MetricsPusher;
+use consts::{NUM_THREADS, STACK_SIZE};
+use futures::future::join_all;
+use std::time::{SystemTime, UNIX_EPOCH};
+use tokio::runtime::{Builder, Runtime};
+
+async fn test_flows(runtime: &Runtime, network_name: NetworkName) -> Result<()> {
+    let run_id = SystemTime::now()
+        .duration_since(UNIX_EPOCH)?
+        .as_secs()
+        .to_string();
+    info!(
+        "----- STARTING TESTS FOR {} WITH RUN ID {} -----",
+        network_name.to_string(),
+        run_id
+    );
+
+    // Flow 1: New account
+    let test_time = run_id.clone();
+    let handle_newaccount = runtime.spawn(async move {
+        TestName::NewAccount.run(network_name, &test_time).await;
+    });
+
+    // Flow 2: Coin transfer
+    let test_time = run_id.clone();
+    let handle_cointransfer = runtime.spawn(async move {
+        TestName::CoinTransfer.run(network_name, &test_time).await;
+    });
+
+    // Flow 3: NFT transfer
+    let test_time = run_id.clone();
+    let handle_nfttransfer = runtime.spawn(async move {
+        TestName::NftTransfer.run(network_name, &test_time).await;
+    });
+
+    // Flow 4: Publishing module
+    let test_time = run_id.clone();
+    let handle_publishmodule = runtime.spawn(async move {
+        TestName::PublishModule.run(network_name, &test_time).await;
+    });
```

Comment:
Not sure if i read these right. Are these running synchronously?

Reply:
They are supposed to run in parallel